### PR TITLE
Adding Maintainer Mountaineer

### PR DESCRIPTION
**Pull request by RichardLitt**
And just pointing to my blog where I talk about donating funds. If my single person LLC has a strategy, it is ad hoc investment in people I run across on GitHub.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -119,6 +119,7 @@ <h2>Who?</h2>
             <li><a title="Frontend Masters" href="https://blog.opencollective.com/frontend-masters/"><img src="./logos/frontend-masters.svg" alt="Frontend Masters logo"></a></li>
             <li><a title="Indeed" href="https://github.com/indeedeng/FOSS-Contributor-Fund"><img src="./logos/indeed.svg" alt="Indeed logo"></a></li>
             <li><a title="Microsoft" href="https://opensource.microsoft.com/ecosystem/"><img src="./logos/microsoft.png" alt="Microsoft logo"></a></li>
+            <li><a title="Maintainer Mountainer" href="https://www.burntfen.com/2023-11-30/sponsoring-open-source-projects"><img src="./logos/maintainer.svg" alt="Maintainer Mountaineer logo" /></a></li>
```
Comment (chadwhitacre) [inline index.html]:
```suggestion
            <li><a title="Maintainer Mountaineer" href="https://www.burntfen.com/2023-11-30/sponsoring-open-source-projects"><img src="./logos/maintainer-mountaineer.svg" alt="Maintainer Mountaineer logo" /></a></li>
```

**Comment by chadwhitacre** [comment]
Thanks! Left a suggestion ... also can we alphabetize? Should come between Indeed and Microsoft, no?

**Comment by RichardLitt** [comment]
Done! Also added a p.s. to that blog post explaining my strategy. :)

**Comment by chadwhitacre** [comment]
Merged, welcome aboard! 😁 💃

**Comment by RichardLitt** [comment]
Woohoo!

**Pull request merged**